### PR TITLE
Updated hugo bins and added makefile

### PR DESCRIPTION
**Pull request by alexmanno**

### CODE REVIEW

File: deploy [hunk 1]
```diff
@@ -1,223 +0,0 @@
-#!/usr/bin/env bash
```
Comment (Jean85) [inline deploy:1]: Why isn't this considered a move?

Comment (alexmanno) [reply deploy:1]: IDK probably git don't recognize changes in this case

**Comment by salvatorecordiano** [comment]
@alexmanno now https://github.com/facile-it/facile-it.github.io/issues/28 it's merged. Please, take care of it

**Comment by salvatorecordiano** [comment]
This issue was closed here https://github.com/facile-it/facile-it.github.io/pull/30

**Pull request closed without merging**